main.py

```
@@ -1,75 +1,22 @@
-# 🎯 Entry point
-async def main():
-    core = OrionCore()
-    await core.run()
-
-if __name__ == "__main__":
-    loop = asyncio.get_event_loop()
+from Backend.STT import FastNaturalSpeechRecognition
 
-    # Create and run the main Orion task
-    main_task = loop.create_task(main())
+async def voice_loop():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)